HOWTO

```diff
@@ -299,6 +299,10 @@ Command line options
 	Use the directory specified by `path` for generated state files instead
 	of the current working directory.
 
+.. option:: --latency-stats=nr
+
+	Change default latency stats number.
+
```

Comment:
In the documentation could you say what the default number of buckets is? Could you also add a note to the CSV documentation warning that the number of buckets may vary?

Reply:
Updated the description for your comments. Thank you.

Reply:
Ah I meant:
- https://github.com/axboe/fio/blob/master/HOWTO#L4048-L4054
- https://github.com/axboe/fio/blob/master/fio.1#L3751-L3768
- https://github.com/axboe/fio/blob/master/HOWTO#L4082-L4087
- https://github.com/axboe/fio/blob/master/fio.1#L3810-L3816

I think just rewrite the description to something like the following:
```
stat: make latency buckets configurable

Allow the user to set how many latency buckets there are at the finest 
granularity (when unset the default is 10). The number of larger granularity 
buckets is calculated as the finest + 2.
```

Out of interest could this cause errors because the system timer doesn't offer a granularity that low? I'm not sure what the smallest piece of time your typical server can measure is (see https://stackoverflow.com/questions/6498972/faster-equivalent-of-gettimeofday/ for some overheads reading various clocks)...

Reply:
> Ah I meant:
> 
> * https://github.com/axboe/fio/blob/master/HOWTO#L4048-L4054
> * https://github.com/axboe/fio/blob/master/fio.1#L3751-L3768
> * https://github.com/axboe/fio/blob/master/HOWTO#L4082-L4087
> * https://github.com/axboe/fio/blob/master/fio.1#L3810-L3816

Noted so updated the documentation again.

> I think just rewrite the description to something like the following:
> 
> ```
> stat: make latency buckets configurable
> 
> Allow the user to set how many latency buckets there are at the finest 
> granularity (when unset the default is 10). The number of larger granularity 
> buckets is calculated as the finest + 2.
> ```

Yes updated the description also.

> Out of interest could this cause errors because the system timer doesn't offer a granularity that low? I'm not sure what the smallest piece of time your typical server can measure is (see https://stackoverflow.com/questions/6498972/faster-equivalent-of-gettimeofday/ for some overheads reading various clocks)...

Currently the minimum latency bucket is 2 nanosecond or lower so it should be worked the changes I think.

Regards,
Ikegami